lib/OpenQA/Setup.pm

```diff
@@ -35,26 +35,27 @@ sub read_config {
     my $app      = shift;
     my %defaults = (
         global => {
-            appname                     => 'openQA',
-            base_url                    => undef,
-            branding                    => 'openSUSE',
-            download_domains            => undef,
-            suse_mirror                 => undef,
-            scm                         => undef,
-            hsts                        => 365,
-            audit_enabled               => 1,
-            max_rss_limit               => 0,
-            profiling_enabled           => 0,
-            monitoring_enabled          => 0,
-            plugins                     => undef,
-            hide_asset_types            => 'repo',
-            recognized_referers         => '',
-            changelog_file              => '/usr/share/openqa/public/Changelog',
-            job_investigate_ignore      => '"(JOBTOKEN|NAME)"',
-            job_investigate_git_timeout => 20,
-            worker_timeout              => DEFAULT_WORKER_TIMEOUT,
-            search_results_limit        => 50000,
-            auto_clone_regex            => '^cache failure: ',
+            appname                       => 'openQA',
+            base_url                      => undef,
+            branding                      => 'openSUSE',
+            download_domains              => undef,
+            suse_mirror                   => undef,
+            scm                           => undef,
+            hsts                          => 365,
+            audit_enabled                 => 1,
+            max_rss_limit                 => 0,
+            profiling_enabled             => 0,
+            monitoring_enabled            => 0,
+            plugins                       => undef,
+            hide_asset_types              => 'repo',
+            recognized_referers           => '',
+            changelog_file                => '/usr/share/openqa/public/Changelog',
+            job_investigate_ignore        => '"(JOBTOKEN|NAME)"',
+            job_investigate_git_timeout   => 20,
+            worker_timeout                => DEFAULT_WORKER_TIMEOUT,
+            search_results_limit          => 50000,
+            auto_clone_regex              => '^cache failure: ',
+            influxdb_minion_job_blocklist => undef,
```

Comment:
I only add `influxdb_minion_job_blocklist => undef`

Reply:
Maybe influxdb deserves is own section although we only have one setting so far. It would also be nice to extend our sample/default config file. The name of the key should maybe indicate that this is about failed jobs.